js/util/browser/canvas.js

```diff
@@ -29,6 +29,8 @@ Canvas.prototype.resize = function(width, height) {
     this.canvas.style.height = height + 'px';
 };
 
+// If you update this object, you must also update mapbox-gl-js-supported
+// https://github.com/mapbox/mapbox-gl-js-supported/blob/gh-pages/index.js#L94
```

Comment:
I felt funny about having `mapbox-gl-js-supported` "own" this data. 

I agree that moving it to`mapbox-gl-js-supported`  carries rather significant advantage. I will adjust my approach.